protocol/daemons/server/types/liquidations/daemon_liquidation_info.go

```diff
@@ -0,0 +1,117 @@
+package types
+
+import (
+	"sync"
+
+	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
+)
+
+// DaemonLiquidationInfo maintains the list of subaccount ids to be liquidated
+// in the next block. Methods are goroutine safe.
+type DaemonLiquidationInfo struct {
+	sync.Mutex                                       // lock
+	blockHeight               uint32                 // block height of the last update
+	liquidatableSubaccountIds []satypes.SubaccountId // liquidatable subaccount ids
+	negativeTncSubaccountIds  []satypes.SubaccountId // negative total net collateral subaccount ids
+	subaccountsWithPositions  map[uint32]clobtypes.SubaccountOpenPositionInfo
```

Comment:
Suggestion: less error prone to have values of `subaccountsWithPositions` be a pointer? Cause default value of `clobtypes.SubaccountOpenPositionInfo` is technically valid (perp ID of 0, no open positions)